lang/elixir/Portfile

```
@@ -3,14 +3,14 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        elixir-lang elixir 1.13.3 v
+github.setup        elixir-lang elixir 1.13.4 v
 github.tarball_from archive
 epoch               1
 categories          lang
 platforms           darwin
 supported_archs     noarch
 license             Apache-2
-maintainers         {@milmazz milmazz.uno:me} {ciserlohn @ci42}
+maintainers         {@milmazz milmazz.uno:me} {ciserlohn @ci42} {gmail.com:secsaba @secsaba}
```

Please also add `openmaintainer` as well, so that even non-maintainers can make minor updates:

```suggestion
maintainers         {@milmazz milmazz.uno:me} {ciserlohn @ci42} {gmail.com:secsaba @secsaba} \
                    openmaintainer
```